data/help.txt

```diff
@@ -97,6 +97,35 @@ help "multiple ship controls"
 	`Escorts can also be ordered with the mouse. Right-clicking on an empty point in space will order escorts to fly to and hold position at that point, while right-clicking on a hostile ship will order your escorts to focus fire on it, and right-clicking on a friendly target will cause them to follow it.`
 	`Escorts can be saved into control groups by pressing Control+number key. All selected escorts will then be bound to that number key, selecting the group when pressed.`
 
+help "fleet asteroid mining"
+	`With an asteroid scanner and a fleet, you can direct your fleet to assist with mining operations and collect mined resources.`
+	`If your escort or fighter has no weapons but has cargo space, then it will chase after mined resources instead of mining.`
+
+help "fleet asteroid mining shortcuts"
+	`Shortcuts for asteroid targeting:`
+	`	<Select nearest asteroid> will select the nearest asteroid with asteroid scanner.`
+	`	<Fleet: Fight my target> will focus your fleet on mining a targeted asteroid (chase and mine).`
+	`	<Fleet: Harvest Flotsam> will toggle fleet harvest flotsam.`
+	``
+	`Alternate targeting:`
+	`	Left click - will target an asteroid without mining it.`
+	`	Right click - will focus your fleet to chase an asteroid and mine it.`
+
+help "fleet harvest tutorial"
+	`If you have an asteroid scanner, <Select nearest asteroid> key will target asteroids for fleet mining.`
+	``
+	`<Fleet: Harvest Flotsam> key is a shortcut for your fleet to harvest discarded cargo and mined asteroids. Press <Fleet: Harvest Flotsam> multiple times to toggle harvest mode on and off for your fleet.`
```

Comment:
Looking through the code it isn't redundant.  They're separate tutorials for different use cases.

Use cases:

* `fleet asteroid mining` and `fleet asteroid mining shortcuts` 2-page tutorial.  This tells a player how to fully utilize all of their shortcuts for fleet mining now that they've obtained an asteroid scanner and have a fleet.  This would only occur if a player decides to actually install an asteroid scanner while they have a fleet.
* `fleet harvest tutorial` triggers when a player gets their first escort (nothing to do with mining).  It explains to the player that they can order their ship to gather flotsam e.g. cargo discarded by other ships during space fights.

Another way to think of `fleet harvest tutorial` is that it introduces the player to fleet harvest  I think I will reword this to be more clear.

Reply:
I updated this tutorial